Makefile

```diff
@@ -1,194 +1,59 @@
 SHELL = /bin/bash -eo pipefail
-
-AWS_FOLDER = .aws
-DOCKER_IMAGE_NAME = observability/apm-lambda-extension
-DOCKER_REGISTRY = docker.elastic.co
-AGENT_VERSION = $(shell echo $${BRANCH_NAME} | cut -f 2 -d 'v')
+export DOCKER_IMAGE_NAME = observability/apm-lambda-extension
+export DOCKER_REGISTRY = docker.elastic.co
 
 # Add support for SOURCE_DATE_EPOCH and reproducble buils
 # See https://reproducible-builds.org/specs/source-date-epoch/
 SOURCE_DATE_EPOCH ?= $(shell git log -1 --pretty=%ct)
 DATE_FMT = +%Y%m%d%H%M.%S
+DATE_FMT_RFC_3339 = +%Y-%m-%dT%H:%M:%SZ
 # Fallback mechanism to support other systems:
 # 1. 'date -d': Busybox and GNU coreutils.
 # 2. 'date -r': BSD date. It does not support '-d'.
-BUILD_DATE = $(shell date -u -d "@${SOURCE_DATE_EPOCH}" "${DATE_FMT}" 2>/dev/null || date -u -r "${SOURCE_DATE_EPOCH}" "${DATE_FMT}")
+export BUILD_DATE = $(shell date -u -d "@${SOURCE_DATE_EPOCH}" "${DATE_FMT}" 2>/dev/null || date -u -r "${SOURCE_DATE_EPOCH}" "${DATE_FMT}")
+export BUILD_DATE_RFC_3339 = $(shell date -u -d "@${SOURCE_DATE_EPOCH}" "${DATE_FMT_RFC_3339}" 2>/dev/null || date -u -r "${SOURCE_DATE_EPOCH}" "${DATE_FMT_RFC_3339}")
```

Comment:
**issue**: `$(shell ...)` is a make variable. If you convert these to env variable you need to remove the `shell`. This should fix the error:

```
date: '': No such file or directory
date: '': No such file or directory
date: '': No such file or directory
date: '': No such file or directory
```

Reply:
I actually got rid of this and used the goreleaser variable `.CommitTimestamp` instead.

https://github.com/elastic/apm-aws-lambda/pull/362/commits/ccb7116483e31cb1ffac3080a25964ebe4b84f95

Reply:
Note: that means losing support for `SOURCE_DATE_EPOCH` but it should be fine since it's not mandatory and the commit timestamp should not change